SU2_CFD/src/output/COutput.cpp

```diff
@@ -1403,7 +1403,7 @@ void COutput::CheckHistoryOutput(){
   }
 
   if (rank == MASTER_NODE){
-    if(wndConvFields.size() == 0){
+    if(convFields.size() == 0){
```

Comment:
`convFields.empty()` always prefer text to numbers, makes the code more expressive.

Reply:
Okay, done :)